File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Location.java

package restaurant.sa.com.sarestaurant.appview.restaurant.model;


import com.google.gson.annotations.Expose;

@SuppressWarnings("unused")
public class Location {

    @Expose
    private Double lat;
    @Expose
    private Double lng;

    public Double getLat() {
        return lat;
    }

    public void setLat(Double lat) {
        this.lat = lat;
    }

    public Double getLng() {
        return lng;
    }

    public void setLng(Double lng) {
        this.lng = lng;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Location.kt
package restaurant.sa.com.sarestaurant.appview.restaurant.model


import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
@Entity(tableName = "Location")
class Location(@ColumnInfo(name = "lat") var lat: Double?,
               @ColumnInfo(name = "lng") var lng: Double?)
